docs/tools/vega-wallet/desktop-app/latest/getting-started.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Get started with Vega Wallet
+hide_title: false
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Vega Wallet is available to use as a desktop application. Set-up takes less than 5 minutes, from downloading the software to connecting to a Vega dApp.
+
+## Download and start the Vega Wallet desktop app
+
+### Download, save and open app
+#### 1. Download and save the file 
+
+Get the latest version of the Vega Wallet desktop app from the [GitHub releases page](https://github.com/vegaprotocol/vegawallet-desktop/releases). 
+
+Keep track of where you've saved the file, because you will need to know where to look for it to use it.
+
+Click through the tabs below to find out which download file you need. 
+
+<Tabs groupId="operating-systems">
+<TabItem value="windows" label="Windows">
+
+Download `vegawallet-windows-amd64.zip`
+
+</TabItem>
+<TabItem value="mac" label="MacOS">
+
+Download `vegawallet-darwin-amd64.zip`
+
+For Macs with an M1 processor (released since November 2020): 
+
+Download `vegawallet-darwin-arm64.zip`
+  
+</TabItem>
+
+<TabItem value="linux" label="Linux">
+
+Download `vegawallet-linux-amd64.zip`
+</TabItem>
+</Tabs>
+
+#### 2. Extract the file
+Click on the compressed folder to get access to the app. 
+
+#### 3. Start the app 
+Double-click on the Vega Wallet desktop app icon to start the desktop app. 
+
+## Generate new wallet
+If you don't already have a Vega wallet, create a new wallet. 
+
+1. Open the app
+2. Click on "Create new wallet" 
+3. Enter a wallet name of your choice, and a passphrase. 
+4. Your first keypair will be created for you. You can create more keys for that wallet but you only need one to start interacting with Vega. 
+5. Choose a Vega network to connect to: Click on the arrow in the bottom bar of the app. 
+
+## Restore existing wallet
+If you already have a Vega wallet with keys, recover the wallet using your recovery phrase. 
+
+1. Open the app. 
+2. Click on "Use recovery phrase"
+3. Enter your recovery phrase, exactly in the order that it was first presented to you. 
+4. Choose your wallet version (1 or 2). 
+5. Choose a Vega network to connect to: Click on the arrow in the bottom bar of the app. 
+
+## Troubleshooting
+The Vega Wallet desktop app does not yet have the full functionality of the CLI wallet app.
+
+### Not supported
+To do any of the following, you will need to use the **[CLI wallet](docs/tools/vega-wallet/cli-wallet/)**:
```

Comment:
For consistency:

```suggestion
To do any of the following, you will need to use the **[wallet CLI](docs/tools/vega-wallet/cli-wallet/)**:
```

Sometimes we have "CLI wallet" et "wallet CLI".

Reply:
I try to use CLI wallet for the app (and generally in the docs) - is there somewhere you saw "wallet CLI", as I'd probably change that one instead